components/button/Button.styles.tsx

```diff
@@ -1,7 +1,22 @@
 import styled from '@emotion/styled'
 
 const StyledButton = styled.button`
-    padding: 10px;
+    padding: 10px 20px;
+    text-transform: uppercase;
+    background-color: var(--Secondary-brand-color);
+    border: 2px solid var(--Off-brand-color);
+    border-radius: 5px;
+    color: var(--Off-brand-color);
+    font-weight: 800;
+    transition: 0.3s ease-in-out;
+
+    &:hover {
+        background-color: var(--Primary-accent-color);
+        border: 2px solid var(--Primary-accent-color);
+        border-radius: 5px;
+        color: var(--Primary-brand-color);
+        font-weight: 800;
```

Comment:
`cursor: pointer`

Reply:
this button needs a rework . hate how it looks and functions . i play later tho in a Fix the Ugly Button  ticket